pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -134,3 +179,37 @@ func (q *QueryReconciler) runListRestorePoints(ctx context.Context, _ *ctrl.Requ
 	return vrpqstatus.UpdateConditionAndState(ctx, q.VRec.Client, q.VRec.Log, q.Vrpq,
 		v1.MakeCondition(vapi.QueryComplete, metav1.ConditionTrue, "Completed"), stateSuccessQuery)
 }
+
+// makeDispatcher will create a Dispatcher object based on the feature flags set.
+func (q *QueryReconciler) makeDispatcher(log logr.Logger, vdb *v1.VerticaDB, prunner cmds.PodRunner,
+	passwd string) vadmin.Dispatcher {
+	if vmeta.UseVClusterOps(vdb.Annotations) {
+		return vadmin.MakeVClusterOps(log, vdb, q.VRec.GetClient(), passwd, q.VRec, vadmin.SetupVClusterOps)
+	}
+	return vadmin.MakeAdmintools(log, vdb, prunner, q.VRec, q.OpCfg.DevMode)
+}
+
+// GetSuperuserPassword returns the superuser password if it has been provided
+func (q *QueryReconciler) GetSuperuserPassword(ctx context.Context, log logr.Logger,
```

Comment:
I think we can remove this because we aren't going to need the dbadmin password.